src/main/java/seedu/address/logic/commands/AddRestaurantNoteCommand.java

```diff
@@ -41,9 +42,9 @@
     public static final String MESSAGE_EMPTY = "No note added to Restaurant: %1$s";
 
     private final Index index;
-    private final Note recommendedFood;
-    private final Note goodFood;
-    private final Note badFood;
+    private final ArrayList<Note> recommendedFood;
```

Comment:
Great that we changed it so that we can allow for people to append multiple notes.